src/lib/storage/create_iterable_from_segment.hpp

```diff
@@ -28,7 +28,7 @@ class ReferenceSegmentIterable;
  * @{
  */
 
-template <typename T, bool EraseSegmentType = HYRISE_DEBUG>
+template <typename T, bool EraseSegmentType = HYRISE_DEBUG, bool EraseReferencedSegmentType = false /*ignored*/>
```

Comment:
a boolean parameter. Even worse, a boolean parameter for six(!) functions that are not concerned at all with whatever this boolean parameter is concerned with. 
There has to be a way around this, pretty please? :fearful:

Reply:
Yes, but I have to copy two lines. See cae829dd6